locales/en-US/editor.properties

```diff
@@ -193,6 +193,14 @@ AUDIO_OPTION_AUTOPLAY=Start playing the audio automatically
 AUDIO_OPTION_LOOPING=Restart the audio when it is over
 AUDIO_OPTION_DISABLE_AUDIO=Start audio player muted
 
+# Binary Viewer
+BINARY_FILE_TITLE=Non-editable File
+BINARY_FILE_CANNOT_EDIT=I'm not sure how to edit this file!
```

Comment:
I don't love the use of "I'm" here.  What about the third-person "Thimble doesn't know how to edit this file!"?